draft-ietf-tls-subcerts.md

```diff
@@ -217,7 +218,7 @@ Client            Front-End            Back-End
   |----ClientHello--->|                    |
   |<---ServerHello----|                    |
   |<---Certificate----|                    |
-  |                   |<-------LURK------->|
+  |                   |<-----PKCS#11------>|
```

Comment:
maybe we can call this remote signature and cite the keyless ssl paper instead

Reply:
Yeah, good idea.

Reply:
Fixed.